src/components/nav.tsx

```diff
@@ -127,26 +131,24 @@ export function Nav({ isCollapsed, links, supportLinks }: NavProps) {
                 </Link>
               ),
             )}
-            {!isCollapsed && (
-            <div className="flex items-center gap-2 ml-[1.5px]">
-              <Avatar>
-                <AvatarImage src={user.imageUrl} alt={user.fullName ?? undefined} />
-                <AvatarFallback>{user.fullName}</AvatarFallback>
-              </Avatar>
-              <div>
-                <p className="text-xs">{user.fullName}</p>
-                <p className="text-xs">{user.primaryEmailAddress?.emailAddress}</p>
-              </div>
-            </div>
-            )}
-            {isCollapsed && (
             <div className="flex items-center gap-2">
-              <Avatar>
-                <AvatarImage src={user.imageUrl} alt={user.fullName ?? undefined} />
-                <AvatarFallback>{user.fullName}</AvatarFallback>
-              </Avatar>
+              <SignedIn>
+                <UserButton
```

Comment:
Instead of using our own Avatar i'm using the built in component clerk for now. Since it has a built in sign out function.